moninag/tests/unittests/registration/tests.py

```diff
@@ -0,0 +1,47 @@
+from django.core.urlresolvers import reverse
+from django.test import Client
+
+# imports for views
+
+from registration.models import CustomUser
+from django.test import TestCase
+
+# imports for everything
+
+
+# models test
```

Comment:
1.  This module should have a proper name (view tests or model tests) since its Unit Tests and we should have separate units tested.
 
2. Provide docstring for each test so we can understand test aim easily, without diving into code 🦊 

3. If this module (and view tests module) isn't finished i guess its a good idea to detach it from this "UI" pull request